core/templates/pages/contributor-dashboard-page/modal-templates/translation-modal.component.ts

```diff
@@ -189,6 +217,32 @@ export class TranslationModalComponent {
     };
   }
 
+  ngAfterViewInit(): void {
+    this.computePanelOverflowState();
+  }
+
+  ngAfterViewChecked(): void {
+    this.computeTranslationEditorOverflowState();
+  }
+
+  computeTranslationEditorOverflowState(): void {
+    const windowHeight = this.wds.getHeight();
+    const heightLimit = windowHeight * (this.cutoff_height) / 100;
+
+    this.isTranslationOverflowing = (
+      this.translationContainer?.nativeElement.offsetHeight >=
+        heightLimit
+    );
+  }
+
+  computePanelOverflowState(): void {
+    setTimeout(() => {
+      this.isContentOverflowing = (
+        this.contentPanel?.elementRef.nativeElement.offsetHeight >
+        this.contentContainer?.nativeElement.offsetHeight);
+    }, 500);
```

Comment:
Can you add a comment here explaining how this value was chosen as explained in https://github.com/oppia/oppia/pull/15956#discussion_r959680473 so that future readers know?

Reply:
Done

Reply:
Thanks.